catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -99,7 +99,163 @@
+                fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/vnd.ms-powerpoint", fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put(
+                "application/vnd.openxmlformats-officedocument.presentationml.presentation",
+                fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put(
+                "application/vnd.openxmlformats-officedocument.presentationml.slideshow",
+                fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put(
+                "application/vnd.openxmlformats-officedocument.presentationml.template",
+                fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/vnd.ms-powerpoint.addin.macroenabled.12",
+                fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/vnd.ms-powerpoint.slideshow.macroenabled.12",
+                fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/vnd.ms-powerpoint.slide.macroenabled.12",
+                fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/vnd.ms-powerpoint.template.macroenabled.12",
+                fallbackPowerpointMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/msword", fallbackOfficeDocMetacardType);
+
+        mimeTypeToMetacardTypeMap.put(
+                "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+                fallbackOfficeDocMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/vnd.ms-word", fallbackOfficeDocMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/vnd.ms-word.document.macroenabled.12",
+                fallbackOfficeDocMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/vnd.ms-word.template.macroenabled.12",
+                fallbackOfficeDocMetacardType);
+
+        mimeTypeToMetacardTypeMap.put(
+                "application/vnd.openxmlformats-officedocument.wordprocessingml.template",
+                fallbackOfficeDocMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/pdf", fallbackPdfMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("video/mpeg", fallbackMpegMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("audio/mpeg", fallbackMpegMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("video/mpeg4-generic", fallbackMpegMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("application/mp4", fallbackMp4MetacardType);
+
+        mimeTypeToMetacardTypeMap.put("audio/mp4", fallbackMp4MetacardType);
+
+        mimeTypeToMetacardTypeMap.put("video/mp4", fallbackMp4MetacardType);
+
+        mimeTypeToMetacardTypeMap.put("image/jpeg", fallbackJpegMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("video/jpeg", fallbackJpegMetacardType);
+
+        mimeTypeToMetacardTypeMap.put("video/jpeg2000", fallbackJpegMetacardType);
```

Comment:
Looks like an earlier version of tika treated .jpm’s as the mime type “video/jpeg2000” but this has since been changed. The correct mappings for the various extensions of jpeg2000 in the version of tika we use should be:
.jp2 : image/jp2
.jpf, .jpx : image/jpx
.jpm : image/jpm
.mj2 : video/mj2
 I will verify that the rest of the mappings to make sure they are checked against the version of tika we use.